src/lib/models/workflow-execution.ts

```diff
@@ -38,7 +37,8 @@ export const toWorkflowExecution = (
   const historyEvents = response.workflowExecutionInfo.historyLength;
   const url = `/workflows/${id}/${runId}`;
   const taskQueue = response?.executionConfig?.taskQueue?.name;
-  const pendingActivities = response.pendingActivities || [];
+  const pendingActivities: PendingActivity[] =
+    (response.pendingActivities as unknown as PendingActivity[]) || [];
```

Comment:
We need to do this because the type that was generated from the protobuf is _not_ what the API actually produces. This aligns it with the _actual_ values returned from the API.